recipe/meta.yaml

```diff
@@ -65,9 +65,6 @@ test:
     {% set tests_to_skip = tests_to_skip + " or test_fsspec_memory_list" %}
     {% set tests_to_skip = tests_to_skip + " or test_elastic_training_dl1_backend_gloo" %}
     {% set tests_to_skip = tests_to_skip + " or test_elastic_training_dl2_backend_gloo" %}
-    # fails because fsspec is not available (AWS S3 stuff)
-    {% set tests_to_skip = tests_to_skip + " or test_fsspec_io_iterdatapipe" %}
-    {% set tests_to_skip = tests_to_skip + " or test_s3_io_iterdatapipe" %}
```

Comment:
Continuing on from https://github.com/conda-forge/torchdata-feedstock/pull/18#discussion_r1405505811, it seems like these tests are still failing on Linux (but not on macOS), even with a new AWS SDK version. I've tested the torchdata-0.7.1-py311ha8bf654_0 build locally to access s3 files, and it works fine to list/download files from an s3 bucket, so not sure why these tests are failing here.

Reply:
I think it's OK to revert that last commit for now (or rebase it out). It could be a lot of things (e.g. around the call to the AWS cli and its expectations), but IMO it's not necessary to investigate at the moment.

Reply:
Sounds good. I've reverted the change, amd can merge it once the CI passes.